_raml/apps/carts/carts.raml

```diff
@@ -26,6 +26,37 @@ post:
             example: !include examples/get_carts_cartid_response.json
             schema: !include schema/get_carts_cartid.schema.json
 
+  /coupon:
+    displayName: Coupon
+    post:
+      description: Creates a coupon code for a cart which results in creating a couponLineItem and a link to remove the coupon.
```

Comment:
All post request descriptions start with create... 
Why can't we apply that for this one, too? Change it to your suggestion, but I'm not happy with it.